en/frames_vtol/vtol_tiltrotor_omp_hobby_zmo_fpv.md

```diff
@@ -0,0 +1,368 @@
+
+   ![LTE antenna 2 and RC receiver](../../assets/airframes/vtol/omp_hobby_zmo_fpv/lte-antenna-02.jpg)
+
+## Software Setup
+
+### Select Airframe
+
+1. Open QGC, select the **Q** icon, and then select **Vehicle Setup**.
+1. Select the [Airframe](https://docs.px4.io/main/en/config/airframe.html) tab
+1. Select [Generic Tiltrotor VTOL](../airframes/airframe_reference.md#vtol_vtol_tiltrotor_generic_tiltrotor_vtol) from the _VTOL Tiltrotor_ group, and then and click **Apply and Restart**.
+
+### Load Parameters File
+
+Next we load a [parameter file](https://github.com/PX4/PX4-user_guide/raw/main/assets/airframes/vtol/omp_hobby_zmo_fpv/OMP-Hobby-ZMO.params) that contains parameters that define the frame geometry, output mappings, and tuning values — so you don't have to!
+If you have followed the wiring instructions for the motors you probably won't need to do much further configuration other than sensor calibration and fixing the trims.
+
+To load the file:
+
+1. Download the [parameter file](https://github.com/PX4/PX4-user_guide/raw/main/assets/airframes/vtol/omp_hobby_zmo_fpv/OMP-Hobby-ZMO.params).
+1. Select the [Parameters](../advanced_config/parameters.md#finding-updating-parameters) tab and then click on **Tools** in the top right corner.
+1. Select **Load from file** and then choose the `OMP-Hobby-ZMO.params` file you just downloaded.
+1. Reboot the vehicle.
+
+### Sensor Calibration
+
+First make sure to set the [correct orientation of the flight controller](../config/flight_controller_orientation.md).
+This should be the default (`ROTATION_NONE`).
+
+Then calibrate the main sensors:
+
+- [Compass](../config/compass.md)
+- [Gyroscope](../config/gyroscope.md)
+- [Accelerometer](../config/accelerometer.md)
+- [Airspeed](../config/airspeed.md)
+
+### RC-Setup
+
+[Calibrate your RC Controller](../config/radio.md) and setup the [flight mode switches](../config/flight_mode.md).
+
+We recommend you assign RC switches for the set of modes defined in [Flight Mode Configuration > What Flight Modes and Switches Should I Set?](../config/flight_mode.md#what-flight-modes-and-switches-should-i-set).
+In particular you should assign a _VTOL Transition Switch_, _Kill Switch_, and a switch to select [Stabilized mode](../flight_modes_fw/stabilized.md) and [Position mode](../flight_modes_fw/position.md).
+
+### Actuator Setup
+
+:::warning
+Make sure the props are removed!
+The motors are easy to start in the actuators tab by accident.
+:::
+
+Motors, control surfaces, and other actuators are configured in the QGroundControl [Actuator Configuration & Testing](../config/actuators.md).
+
+The [parameter file](#load-parameters-file) loaded previously means that this screen should already be correctly setup: you just need to adjust the trims for your particular vehicle.
+If motors/servos were connected to different outputs than suggested, you will need to change the output to function mappings in the actuator output section.

```

Comment:
FYI this explains (again) why we're not doing geometry etc, and points out you might need to change wiring.